src/pretix/plugins/stripe/static/pretixplugins/stripe/pretix-stripe.js

```diff
@@ -196,28 +241,47 @@ $(function () {
         return;
 
     if ($("input[name=payment][value=stripe]").is(':checked') || $(".payment-redo-form").length) {
-          pretixstripe.load();
+        pretixstripe.load();
     } else {
         $("input[name=payment]").change(function () {
-            if ($(this).val() === 'stripe') {
+            if (['stripe', 'stripe_sepa_debit'].includes($(this).val())) {
```

Comment:
We have not yet dropped IE11 support for the most fundamental features like "paying for a ticket". `includes` is unfortunately not supported by IE11, use indexOf or a plain ||